### PR TITLE
prettify: sass files in client/account-recovery

### PR DESCRIPTION
**Pull request by flootr**
#### Changes proposed in this Pull Request

* Run `prettier` on `*.scss` files in `client/account-recovery`

#### Testing instructions

* Have a look at the code changes and make sure it's only formatting related changes

### CODE REVIEW

File: client/account-recovery/account-recovery-root/style.scss [hunk 1]
```diff
@@ -1,9 +1,11 @@
+/** @format */
```
Comment (flootr) [inline client/account-recovery/account-recovery-root/style.scss]: apparently these comments get shipped into production css which is not what we want. may revisit with another approach

Comment (frontdevde) [reply client/account-recovery/account-recovery-root/style.scss]: Not ideal but let's finish one pass before we touch everything again 🙈

**Comment by frontdevde** [comment]
Code wise should be good to merge. The `ci/wp-desktop` tests were failing but I assume that's a fluke. Just triggered a rebuild which should fix that.

**Comment by flootr** [comment]
closing as we're stepping back from formatting sass files with prettier

**Pull request closed without merging**